docs/modules/game-object-creation/standard-usage.md

```diff
@@ -0,0 +1,42 @@
+<%(Toc)%>
+# Setup basic spawning
```

Comment:
is it setup or set up? 
I would have gone with set up (as the verb) for this and the next header, but not entirely sure

Reply:
Ah - I think it may be set up

Reply:
it's still setup your